src/components/RichText.tsx

```diff
@@ -29,7 +29,7 @@ export function RichText({
   const [richText, setRichText] = React.useState<RichTextAPI>(() =>
     value instanceof RichTextAPI ? value : new RichTextAPI({text: value}),
   )
-  const styles = [a.leading_normal, style]
+  const styles = [a.leading_snug, flatten(style)]
```

Comment:
Default's to what we use for post text. `leading_normal` (1.5) is too wide for anything but longform, and `tight` (1.15) is better for titles. `snug` (1.3) is a good default.

Previous `RichText` component defaulted to 1.2, which is a little tight imo.